models/demos/t3000/falcon40b/tests/test_falcon_end_to_end.py

```diff
@@ -401,23 +401,25 @@ def run_test_FalconCausalLM_end_to_end(
     (
         ("prefill", 1, 32, 0),
         ("prefill", 2, 32, 0),
-        # ("prefill", 1, 64, 0),
         ("prefill", 1, 128, 0),
-        # ("prefill", 1, 256, 0),
-        # ("prefill", 1, 512, 0),
-        # ("prefill", 1, 1024, 0),
         ("prefill", 1, 2048, 0),
+        # ("prefill", 1, 8192, 0),  # Do not run in CI: takes too long
+        # ("prefill", 1, 16384, 0),
+        # ("prefill", 1, 32768, 0),
+        # ("prefill", 1, 131072, 0),
+        # ("prefill", 1, 262144, 0),
```

Comment:
Could you do this by using the is_ci fixture like here? https://github.com/tenstorrent/tt-metal/pull/10371
So that you don't need to uncomment the variants to run them locally